src/sentry/models/environmentbookmark.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import BaseManager, FlexibleForeignKey, Model, sane_repr
+from sentry.models import Environment
+
+
+class EnvironmentBookmark(Model):
+    """
+    Identifies a bookmark relationship between a user and an environment
+    """
+
+    __include_in_export__ = True
+
+    environment = FlexibleForeignKey(Environment, blank=True, null=True, db_constraint=False)
```

Comment:
Why `blank=True, null=True`?

Reply:
This was copied over from projectbookmark

https://github.com/getsentry/sentry/blob/de6015f2495121c592f02abe28183951e1004aaf/src/sentry/models/projectbookmark.py#L16

I agree doesn't seem like we need this. Assuming that it does cascade delete when the environment is removed.